llbuild.xcodeproj/project.pbxproj

```diff
@@ -3857,6 +3946,60 @@
 			};
 			name = Release;
 		};
+		E0E37CAF2257EFE5001F2B00 /* Debug */ = {
+			isa = XCBuildConfiguration;
+			buildSettings = {
```

Comment:
Delete all the boilerplate build settings that are unnecessary or inherited from the parent level (which is probably all of these).

Reply:
This target will be removed/replaced by a unit test target eventually.